pom.xml

```
@@ -74,7 +74,7 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.289.x</artifactId>
+        <artifactId>bom-2.303.x</artifactId>
         <version>1117.v62a_f6a_01de98</version>
```

Since we're upgrading the minimum required Jenkins version and the matching plugin bom version, shouldn't we also upgrade the version of the plugin bom?
```suggestion
        <artifactId>bom-2.303.x</artifactId>
        <version>1382.v7d694476f340</version>
```

---

You could. It was not necessary for this purpose so I left it at the maintainer’s discretion to do that now or in some independent PR.